src/imp/schannel.rs

```diff
@@ -91,6 +90,16 @@ impl Identity {
                 .into());
             }
         };
+        
+        let mut store = CertStore::open_current_user("RustMy").unwrap();
```

Comment:
.unwrap() will panic for error. Instead CertStore::open_current_user("RustMy")?; This will return the error.